src/main/resources/templates/db_scripts/schema.json

```diff
@@ -79,10 +79,32 @@
       ]
     },
     {
-      "tableName": "acquisitions_units",
+      "tableName": "acquisitions_unit",
       "fromModuleVersion": 6.1,
       "withMetadata": true,
       "populateJsonWithId": true
+    },
+    {
+    "tableName": "acquisitions_unit_membership",
+      "fromModuleVersion": 6.1,
+      "withMetadata": true,
+      "populateJsonWithId": true,
```

Comment:
`populateJsonWithId` should not be used anymore because [PR #131](https://github.com/folio-org/mod-orders-storage/pull/131/files#diff-162ca424ba9ad000edded275876b325b) migrated to RMB `25.0.1` which [does not support this property](https://github.com/folio-org/raml-module-builder/blob/v25.0.1/doc/upgrading.md#version-25)

Reply:
Done